pom.xml

```diff
@@ -92,11 +91,11 @@
 									<groupId>org.bonitasoft.web.page</groupId>
 									<artifactId>page-application-directory</artifactId>
 									<version>${project.version}</version>
+									<classifier>sources</classifier>
 									<type>zip</type>
```

Comment:
Since all those pages are unpacked in the same directory, can they be put inside a single `<executiuon>`?